### PR TITLE
more actions

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/sunpy/issues/5896 
Fixes https://github.com/sunpy/sunpy/issues/5895

### CODE REVIEW

File: .github/workflows/python-test-workflow.yml [hunk 1]
```diff
@@ -31,7 +30,12 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install tox tox-pypi-filter
+    - name: Build docs
+      if: contains(inputs.tox-env, 'build_docs')
```
Comment (nabobalis) [inline .github/workflows/python-test-workflow.yml:34]: This is not pretty but it works as a solution for now.

File: .github/workflows/python-test-workflow.yml [hunk 2]
```diff
@@ -31,7 +30,12 @@ jobs:
...
+      run: |
+        tox -e ${{ inputs.tox-env }}
     - name: Run tests
+      if: startsWith(inputs.tox-env, 'py')
```
Comment (nabobalis) [inline .github/workflows/python-test-workflow.yml:38]: This is not pretty but it works as a solution for now.

**Review by ConorMacBride — APPROVED** [review]

**Comment by dstansby** [comment]
Why did you get rid of the windows and MacOS builds?

**Comment by dstansby** [comment]
Also I'm not sure there's much point having a CRON build given there's no notification in element for it?

**Comment by nabobalis** [comment]
> Why did you get rid of the windows and MacOS builds?

Azure has them?

**Comment by nabobalis** [comment]
> Also I'm not sure there's much point having a CRON build given there's no notification in element for it?

So we can at least check on the odd occasion that the online and docbuild pass on main?

**Comment by dstansby** [comment]
Ah yeah, I forgot about Azure 😆

**Comment by nabobalis** [comment]
> Ah yeah, I forgot about Azure 😆

If we decide to forget about azure fully, then I will add them back.

**Pull request merged**